crops/command_line/crops-renumber.py

```diff
@@ -74,8 +77,14 @@ def main():
                     newstructure = cop.renumber_pdb(seqset[seqname], structure)
                 except (AttributeError, IndexError) as e:
                     logger.warning('Something has gone wrong during renumbering:\n{}'.format(e))
-                    logger.info('Attempting Needleman-Wunsch...')
-                    newstructure = cop.renumber_pdb_needleman(seqset[seqname], structure)
+                    if args.force_alignment:
+                        logger.info('Attempting Needleman-Wunsch...')
+                        newstructure = cop.renumber_pdb_needleman(seqset[seqname], structure)
+                    else:
+                        logger.critical(('Unable to renumber the structure, exiting now. ' +
+                                         'Try again with -f option to force the alignment.'))
```

Comment:
Using `+` to concatenate two strings is not the most efficient way of doing things, particularly when they are hard coded like in this case. Also it adds unnecessary complexity to the code. Why not:
```python
       logger.critical('Unable to renumber the structure, exiting now. ' 
                       'Try again with -f option to force the alignment.')
```
The result is equivalent (prints everything in the same line).